include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_bwd_data_multiple_d_xdl_cshuffle_v1.hpp

```diff
@@ -71,7 +70,8 @@ template <typename GridwiseGemm,
           typename Block2ETileMap,
           typename ComputePtrOffsetOfBatch,
           typename ComputePtrOffsetOfN,
-          bool HasMainKBlockLoop>
+          bool HasMainKBlockLoop,
+          InMemoryDataOperationEnum OutElementOp>
```

Comment:
Is it another element-wise op? Should we move it to other element-wise ops?

Reply:
Hi, no, it is MemoryDataOperation (Set or AtomicAdd)